plugins/com.google.cloud.tools.eclipse.appengine.validation/src/com/google/cloud/tools/eclipse/appengine/validation/VersionQuickAssistProcessor.java

```diff
@@ -17,7 +17,7 @@
 package com.google.cloud.tools.eclipse.appengine.validation;
 
 /**
- * Provides quick assists for appengine-web.xml in the source editor.
+ * Provides quick fixes to remove the version element from appengine-web.xml.
```

Comment:
How about "... fixes to the {@code version} element ..."?

Reply:
done